src/Microsoft.ML.Data/Depricated/Vector/VBufferMathUtils.cs

```diff
@@ -350,7 +350,7 @@ public static void AddMultWithOffset(ref VBuffer<Float> src, Float c, ref VBuffe
         /// Perform in-place scaling of a vector into another vector as
         /// <c><paramref name="dst"/> = <paramref name="src"/> * <paramref name="c"/></c>.
```

Comment:
I wonder why we have this depricated folder. Can you please check if the functions in here have any references?

Reply:
They certainly do. I have no idea why this was put into deprecated, maybe @Ivanidzo4ka knows.

Reply:
It is also spelled as "Depricated" :(

Reply:
as an responsible adult I would totally push my fault on shoulders of others
[https://msdata.visualstudio.com/AlgorithmsAndDataScience/_git/TLC/commit/f03fff36654042146dbebda077356bf45d534127?refName=refs%2Fheads%2FTLC_Before_github&_a=contents&path=%2FCore%2Fsrc%2FTlcMain%2FTlcMain.csproj](https://msdata.visualstudio.com/AlgorithmsAndDataScience/_git/TLC/commit/f03fff36654042146dbebda077356bf45d534127?refName=refs%2Fheads%2FTLC_Before_github&_a=contents&path=%2FCore%2Fsrc%2FTlcMain%2FTlcMain.csproj)

Let's say it's been like this forever.



---
In reply to: [199894576](https://github.com/dotnet/machinelearning/pull/485#discussion_r199894576) [](ancestors = 199894576)

Reply:
Sounds fine.